src/codegen/spirv.zig

```diff
@@ -2007,6 +2007,7 @@ const DeclGen = struct {
             .array_to_slice => try self.airArrayToSlice(inst),
             .slice          => try self.airSlice(inst),
             .aggregate_init => try self.airAggregateInit(inst),
+            .memcpy         => return self.airMemcpy(inst),
```

Comment:
I think i saw memcpy_safe too, can this handle that too?

Reply:
there's no `memcpy_safe`. i think you saw `memset_safe`

Reply:
Ah, oops